src/jit/optimizer.cpp

```diff
@@ -1483,12 +1483,17 @@ void Compiler::optFindNaturalLoops()
 
      */
 
+    bool          mod = false;
     BasicBlock*   head;
     BasicBlock*   top;
     BasicBlock*   bottom;
     BasicBlock*   entry;
     BasicBlock*   exit;
```

Comment:
Done.  I separated `lastExit` and `onlyExit` while at it, which makes it read more clearly I think.